src/Arcus.Security.Providers.AzureKeyVault/KeyVaultSecretProvider.cs

```diff
@@ -397,7 +397,7 @@ await foreach (SecretProperties property in properties)
             Func<SecretClient, Task<TResponse>> interactWithNewClient)
         {
             var isSuccessful = false;
-#if NET6_0
+#if NET6_0 || NET8_0
```

Comment:
I think the conditional define can be removed, as I think the 'else' part was only needed for the .NET Core 3.1 version of Arcus observability ?

Reply:
Yes, that's right! Removed this. Thx!

Reply:
Removed the directives here: https://github.com/arcus-azure/arcus.security/pull/421/commits/d24b66901b7efe25e896f007b5b6a323722a58c4